storage/innobase/xtrabackup/src/backup_mysql.h

```diff
@@ -245,6 +245,11 @@ bool is_fts_index(const std::string &tablespace);
 @return true if tablespace is temporary table. */
 bool is_tmp_table(const std::string &tablespace);
 
+/** Identifies if table is blocked by select - PS compression dictionary.
+@param[in]	tablespace		tablespace
+@return true if tablespace is blocked. */
```

Comment:
am bit confused if we are usin tablespace or table name here?

Reply:
name